### PR TITLE
PM-13236 PasswordHealthReportApplication DB Tables

### PR DESCRIPTION
**Pull request by voommen-livefront**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/PM-13236

## 📔 Objective

Creating database table, view and queries for `dbo.PasswordHealthReportApplications`

## 📸 Screenshots

No screen shots at this time - as these are database changes only.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: src/Sql/dbo/Stored Procedures/PasswordHealthReportApplications_Create.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CREATE OR ALTER PROC dbo.PasswordHealthReportApplications_Create
+    @Id UNIQUEIDENTIFIER OUTPUT,
+    @OrganizationId UNIQUEIDENTIFIER,
+    @Uri nvarchar(max),
+    @CreationDate DATETIME2(7),
+    @RevisionDate DATETIME2(7)
+AS
+    INSERT INTO dbo.PasswordHealthReportApplications ( Id, OrganizationId, Uri, CreationDate, RevisionDate )
```
Comment (rkac-bw) [inline src/Sql/dbo/Stored Procedures/PasswordHealthReportApplications_Create.sql:8]: SET NOCOUNT ON; should be present

**Review by ttalty — DISMISSED** [review]

**Review by rkac-bw — APPROVED** [review]
lgtm

**Review by rkac-bw — APPROVED** [review]
lgtm

**Comment by voommen-livefront** [comment]
@ttalty @rkac-bw FYI: changed the name of the table from `PasswordHealthReportApplications` to `PasswordHealthReportApplication` (singular) - this required that I change the file names too.

**Review by ttalty — APPROVED** [review]

File: src/Core/AdminConsole/Entities/PasswordHealthReportApplication.cs
Comment (ttalty) [inline src/Core/AdminConsole/Entities/PasswordHealthReportApplication.cs:1]: Can you move this file to: src\Core\Tools\Entities so we are the code owners for reviews?

**Review by ttalty — CHANGES_REQUESTED** [review]

**Review by ttalty — APPROVED** [review]
Looks good to go 🚀

**Review by rkac-bw — APPROVED** [review]
LGTM

**Pull request merged**